lib/backend/registrybackend/config_test.go

```diff
@@ -14,13 +14,17 @@
 
 package registrybackend
 
-import "github.com/uber/kraken/lib/backend/registrybackend/security"
+import (
+	"github.com/uber/kraken/lib/backend/registrybackend/security"
+	"github.com/uber/kraken/utils/httputil"
+)
 
 func newTestConfig(addr string) Config {
 	return Config{
 		Address: addr,
 		Security: security.Config{
-			EnableHTTPFallback: true,
+			TLS: httputil.TLSConfig{Server: httputil.X509Pair{Disabled: true}, Client: httputil.X509Pair{Disabled: true}},
+			EnableHTTPFallback: false,
```

Comment:
My understanding is "EnableHTTPFallback: true" will trigger a HTTP request if HTTPS fails. 
Is it not true for these failed tests?